addons/web/static/src/js/views/sample_server.js

```diff
@@ -227,23 +227,25 @@ odoo.define('web.SampleServer', function (require) {
                     return false;
                 case "date":
                 case "datetime": {
-                    const delta = Math.floor((Math.random() - Math.random()) * SampleServer.DATE_DELTA);
+                    const delta = Math.floor(
+                        (this._getRandomFloat() - this._getRandomFloat()) * SampleServer.DATE_DELTA
+                    );
                     const date = new moment().add(delta, "hour").format(
```

Comment:
maybe have a getRandomDate method for consistency?

Reply:
Good point, done